lib/features/children/generosity_challenge/models/day.dart

```diff
@@ -1,29 +1,52 @@
 import 'package:equatable/equatable.dart';
+import 'package:givt_app/features/children/generosity_challenge/models/enums/day_chat_status.dart';
+import 'package:givt_app/features/children/generosity_challenge_chat/chat_scripts/models/chat_script_item.dart';
 
 class Day extends Equatable {
   const Day({
     required this.dateCompleted,
```

Comment:
Is there a reason why we're not using nullable for dateCompleted? I see in several places in the code that an empty string is assigned.

Reply:
afaik we don't have clear rules here,
however my personal prefs is to avoid nullable when possible)